modules/simple-payments/simple-payments.css

```diff
@@ -0,0 +1,21 @@
+body .jetpack-simple-payments__wrapper div.jetpack-simple-payments__purchase-message {
+	display: none;
+}
+
+body .jetpack-simple-payments__wrapper div.jetpack-simple-payments__purchase-message,
+body .jetpack-simple-payments__wrapper div.jetpack-simple-payments__purchase-message a {
+	color: #fff;
+	padding: 1em;
+}
+
+body .jetpack-simple-payments__wrapper div.jetpack-simple-payments__purchase-message.show {
```

Comment:
> The rest should just be fine as they were.

Let's rollback in #7500.

Are we displaying links in the message?

we were adding the emial address as a link. Let's cleaning it as well.